doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -202,7 +202,13 @@ nRF5340 Audio
 nRF Desktop
 -----------
 
-|no_changes_yet_note|
+* Added handling of :c:struct:`hid_report_subscriber_event` to :ref:`nrf_desktop_hid_forward`.
+  Handling the :c:struct:`hid_report_subscriber_event` allows to take into account the maximum number of reports with different ID, which can be processed by the subscriber (:c:member:`hid_report_subscriber_event.params.report_max`).
```

Comment:
```suggestion
  Handling the :c:struct:`hid_report_subscriber_event` allows to take into account the maximum number of reports with different ID, which can be processed by the subscriber (:c:member:`hid_report_subscriber_event.params.report_max`).
```

:c:member:`hid_report_subscriber_event.params.report_max` don't like  to API docs. Should this be :c:member:`hid_report_subscriber_event.report_max` ?

Reply:
Looking at the struct declaration it should be `:c:member:hid_report_subscriber_event.params.report_max`

Reply:
Anyway, doxygen seems to "flatten" the struct. Let's try without `params` then